tsconfig.json

```diff
@@ -7,9 +7,10 @@
       "dom.iterable",
       "ESNext"
     ],
+    "sourceMap": true,
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
This needs to be changed to `true` again, we want to keep code quality standards high.